src/azure-cli/azure/cli/command_modules/acr/import.py

```diff
@@ -86,7 +93,39 @@ def acr_import(cmd,
                                               target_tags=target_tags,
                                               untagged_target_repositories=repository,
                                               mode=ImportMode.force.value if force else ImportMode.no_force.value)
-    return client.import_image(
+    result_poller = client.import_image(
         resource_group_name=resource_group_name,
         registry_name=registry_name,
         parameters=import_parameters)
+
+    return _handle_result(cmd, result_poller, source_registry, source_image, registry)
+
+
+def _handle_result(cmd, result_poller, source_registry, source_image, registry):
+    try:
+        result = LongRunningOperation(cmd.cli_ctx, 'Importing image...')(result_poller)
+    except CLIError as e:
```

Comment:
Can we detect that a user specifies registry twice in validator before the execution of the command?

Reply:
not necessarily, it could be valid and would be noisy in those cases.

Reply:
It could be valid as @nathana1 said, it is possible that an image has the registry loginserver in its name...